### DISCUSSION TITLE
Duda sobre pregunta del informe "Como se implementan en memoria las estructuras usadas."

### DISCUSSION THREAD

**Discussion by lfb1206** [IIC2133-PUC/2024-1#108] [Tarea 1] [answered]
No entiendo bien a que apunta esta pregunta, en el sentido de si debemos relatar la secuencia en la que se reserva memoria y libera para cada estructura, cuanta memoria utilizan o alguna otra explicación. 

Muchas gracias de antemano.

**Comment by benjavicente** [comment, marked as answer]
Los heaps se puede trabajar de 2 maneras en memoria.

Los árboles tienen sus hojas o nodos con ciertos valores, y además, está el arreglo con todos los datos. Esos se tienen que relacionar de alguna manera, lo que se busquen que expliquen (no es necesario poner explicito que cada nodo tiene X cosa, pero la relación con el resto de las estructuras y datos sí).
